examples/platform/nrfconnect/util/include/NFCWidget.h

```diff
@@ -0,0 +1,38 @@
+/*
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
```

Comment:
Is it a common approach in CHIP? Why not use default #ifndef guards?

Reply:
Yes, CHIP prefers `#pragma once` since `#ifndef` guards may introduce name collisions (two headers using the same guard name).

Reply:
Yes, some time ago there was discussion about it on Slack.